README.md

```diff
@@ -1,77 +1,172 @@
+        ]
+    )
 
-TODO add -t help
+    def import_wordpress_data(self, data):
+        # wagtail page model fields
+        self.title = data["title"]
+        self.slug = data["slug"]
+        self.first_published_at = data["first_published_at"]
+        self.last_published_at = data["last_published_at"]
+        self.latest_revision_created_at = data["latest_revision_created_at"]
+        self.search_description = data["search_description"]
 
-### Default import command
+        # debug fields
+        self.wp_post_id = data["wp_post_id"]
+        self.wp_post_type = data["wp_post_type"]
+        self.wp_link = data["wp_link"]
+        self.wp_raw_content = data["wp_raw_content"]
+        self.wp_block_json = data["wp_block_json"]
+        self.wp_processed_content = data["wp_processed_content"]
+        self.wp_normalized_styles = data["wp_normalized_styles"]
 
-The package comes with a default command: 
+        # own model fields
+        self.body = data["body"]
+```
 
-- [import_xml.py](wagtail_wordpress_import/management/commands/import_xml.py)
+## Running the import command
 
-The configuration is set to import all items in the XML file that have a `<wp:post_type>` = `post` or `page`. The command only imports an item that has `<wp:status>` = `draft` or `publish`. By default each item is created using a Page model of PostPage()
+The most basic command would be:
 
-## Changing the import configuration
+```bash
+`python manage.py import_xml path/to/xml/file.xml parent_page_id`
+```
 
-[View Documentation](docs/prefilters.md)
+[parent_page_id] is the id of the page in your wagtail site where pages will be created as child pages. If you don't know what that is then choose to edit the page in wagtail admin and look at the url in your browser e.g. `http://www.domain.com/admin/pages/3/edit/` the number after /admin/pages/`3` is the id to use.
 
-# XML item fields -> Wagtail model field mapping
+Running this command will import all Wordpress 'post' and 'page' types to the 'PostPage' model in and app called 'pages'
 
-TODO 
+### Optional command arguments
 
-[View Documentation](docs/mapping.md)
+- `-m` can be used to specify the Wagtail Page model to use for all imported pages. The default is `PostPage` when it's not specified.
+- `-a` can be used to specify the Wagtail App where you have created your Wagtail Page model. The default is `pages` when it's not specified.
+- `-t` can be used to limit the Wordpress page types to be imported. You can pass in a comma separated string of page types or just a single page type. The default is `page,post` if not specified.
+- `-s` can be used to specify the status of pages you want to import. You can pass in a comma separated string of statuses or just a single status. The default is `publish,draft` if not specified.
 
-# Creating page models
+[View the import command source](wagtail_wordpress_import/management/commands/import_xml.py) to if you need to extend the command.
```

Comment:
**Issue**: I'm not sure what this sentence is intended to say.

Reply:
I think one of my just in case suggestions again. Will remove it.